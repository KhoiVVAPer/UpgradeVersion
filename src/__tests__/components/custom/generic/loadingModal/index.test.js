import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import LoadingModal from '../../../../../components/custom/generic/loadingModal';

const setUp = (props) => {
  const component = shallow(<LoadingModal {...props} />);
  return component;
};

describe('components > custom > elements > LoadingModal', () => {
  let component;
  const props = {};
  beforeEach(() => {
    component = setUp(props);
    return component;
  });

  it('Check snapshot', () => {
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Renders Modal component', () => {
    const input = component.find('Modal').length;
    const output = 1;
    expect(input).toEqual(output);
  });

  it('Renders loading image', () => {
    const input = component.find('Image').length;
    const output = 1;
    expect(input).toEqual(output);
  });
});
