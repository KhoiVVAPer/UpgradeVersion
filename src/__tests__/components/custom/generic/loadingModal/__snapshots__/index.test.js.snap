// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components > custom > elements > LoadingModal Check snapshot 1`] = `
<react-native-mock>
  <react-native-mock>
    <react-native-mock />
  </react-native-mock>
</react-native-mock>
`;
